### ISSUE TITLE
Appimage not show icons 

### ISSUE THREAD

**Issue by pedropaulonet** [Symbian9/azpainter#100]
On Linux POP OS! , azpainter does not show the icons.

![image](https://user-images.githubusercontent.com/19802396/96993962-63001600-1502-11eb-9b56-367ec2f221bd.png)

**Comment by dandv** [comment]
Appimage not work at all.

I found this app via https://appimage.github.io/AzPainter/. No appimage to easily download. After I found the download page for appimage, it says build error.

**Comment by Symbian9** [comment]
> Appimage not work at all.

Yep, this is known issue. AppImage builds of AzPainter is ***not ready for use yet***, see:

- https://github.com/Symbian9/azpainter/wiki/Packaging-status#appimage-build

I need someone to help with AppImage creating, but as for now I may just recommend DO NOT use AzPainter AppImage builds.
